index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <header class="header">
+        <div class="header-contain">
+            <div class="logo">My Logo</div>
+            <div>
+                <img class="vector" src="./assets/img/Union.png" alt="">
+                
+            </div>
+        </div>
+    </header>
+    <section>
+        <div class="main__section">
+            <div>
+            <h1 class="title">I'm M.Zaki <br> glad to see <br> you!</h1>
+            <p class="text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect-btn">LET'S CONNECT</p>
+            <div class="media-link">
+                <div class="twitter">
+                    <img src="./assets/img/Vector-3.png" alt="">
+                </div>
+                <div class="twitter">
+                    <img src="./assets/img/Linkedin icon.png" alt="">
+                </div>
+                <div class="linkedin">
+                    <img src="./assets/img/Vector-1.png" alt="">
+                </div>
+                <div class="medium">
+                    <img src="./assets/img/Vector-2.png" alt="">
+                </div>
+                <div class="github">
+                    <img src="./assets/img/Vector (2).png" alt="">
+                </div>
+                <div class="anglelist"></div>
```

Comment:
- [ ] The social icons should use `ul` tag to group them,  and` li` per icon and each icon should have a link `(a tag)` 😻 . Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. This [Javapoint Tutorial](https://www.javatpoint.com/how-to-add-social-media-icons-in-html) has guidance on the best way to use social icons in HTML 😊 . Kindly implement this to ensure following the semantic HTML.